content/zh-cn/blog/_posts/2024-02-21-prevent-unauthorized-volume-mode-conversion-ga.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: blog
+title: "Kubernetes 1.30: Preventing unauthorized volume mode conversion moves to GA"
+date: 2024-04-30
+slug: prevent-unauthorized-volume-mode-conversion-ga
+---
+
+<!--
+layout: blog
+title: "Kubernetes 1.30：防止未经授权的卷模式转换进阶到 GA"
```

Comment:
This should be line 3.

Reply:
Thank you for your review, but I don't understand what you mean here

Reply:
The translated `title` should not be the one in comment.
We are supposed to translate line 3.

Reply:
Updated, thanks